npm/svelte/rollup.config.js

```diff
@@ -0,0 +1,62 @@
+import ts from 'rollup-plugin-typescript2'
```

Comment:
I wonder if we can reduce this duplication every time by importing from a base (maybe in `mount-utils`?)

Something like


https://github.com/cypress-io/cypress/blob/develop/npm/react18/rollup.config.js#L1

And change the RollupConfig to take some params? Or do you think it's better to make each package completely self contained?

Reply:
I was getting annoyed at the duplication so I refactored them in [6cb860a](https://github.com/cypress-io/cypress/pull/23519/commits/6cb860a690242b6360c6a374365d84e32f481b6c). Let me know what you think!